tools/topology/sof-cml-demux-rt5682-max98357a.m4

```diff
@@ -0,0 +1,60 @@
+#
+# Topology for Cometlake with rt5682 headset on SSP0, max98357a spk on SSP1
+#
+# Adding max98357a spk on SSP1 on top of sof-cml-rt5682.
+#
+
+# Include SOF CML RT5682 Topology
+# This includes topology for RT5682, DMIC and 3 HDMI Pass through pipeline
+include(`sof-cml-demux-rt5682.m4')
+
+DEBUG_START
```

Comment:
What is this? Most of the topologies don't have it.

Reply:
It defined in m4/debug.m4 and used for m4 macros debug.
@juimonen , could you give us more detailed explanations here?

Reply:
@keqiaozhang it's only needed for debugging so it can be removed.

Reply:
thanks @lgirdwood, this debugging is removed.